processes/processes.tex

```diff
@@ -469,25 +512,26 @@ \subsection{Exit statuses}
 
 There is a convention about exit codes.
 If the process exited normally and everything was successful, then a zero should be returned.
-Beyond that, there isn't too many conventions except the ones that you place on yourself.
+Beyond that, there aren't too many conventions except the ones that you place on yourself.
 If you know how the program you spawn is going to interact, you may be able to make more sense of the 256 error codes.
-You could in fact write your program to return \texttt{1} if the program went to stage 1 (like writing to a file) \texttt{2} if it did something else etc... But none of the Unix programs are designed to follow that for simplicity sake.
+You could for example, write your program to return \texttt{1} if the program went to stage 1 (like writing to a file) \texttt{2} if it did something else etc... 
+However, none of the UNIX programs are designed to follow this policy, for the sake of simplicity.
```

Comment:
None is a strong word.

Reply:
I am not sure what the correct answer is -- this was what was already mentioned.

Reply:
Oh yeah. Just say usually. Some programs have specialized values.